content/en/continuous_integration/intelligent_test_runner/python.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Intelligent Test Runner for Python
+kind: documentation
+further_reading:
+    - link: "/continuous_integration/tests"
+      tag: "Documentation"
+      text: "Explore Test Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+{{< callout url="#" btn_hidden="true" >}}Intelligent Test Runner for Python (using `pytest`) in beta.{{< /callout >}}
+
+## Compatibility
+
+Intelligent Test Runner is only supported in the following versions and testing frameworks:
+
+* `pytest>=6.8.0`
+  * From `ddtrace>=1.18.0`.
+  * From `Python>=3.7`.
+  * Requires `coverage>=5.5`.
+
+## Setup
+
+Prior to setting up Intelligent Test Runner, set up [Test Visibility for Python][1]. If you are reporting data through the Agent, use v6.40+/v7.40+.
+
+To enable Intelligent Test Runner, set the following environment variables:
+
+`DD_APPLICATION_KEY` (Required)
```

Comment:
nit: They are equivalent but I think we're moving to `DD_APP_KEY` as used in the code below